portfolio/src/main/webapp/script.js

```diff
@@ -1,28 +1,39 @@
-// Copyright 2019 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//     https://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
+function addRandomQuote() {
+    const quotes =
```

Comment:
Structuring your data this way makes it very hard to maintain / error prone, because it's critical to update matching entries in two independent lists.

This would be much clearer and maintainable with a more complex data structure (a Class, or even just an object with named keys).

```
const quotes = [
 {quote: "spes consilium non est", source: "Anonymous"},
 {quote: "...",  source: "..."}
];
```
You can then be sure that both values are always handled/updated together, and that there's no risk that the lists will have e.g. different lengths or that future edits are made in the wrong position.


```
const text = quotes[index].quote;
const name = quotes[index].name;
```